### DISCUSSION TITLE
h11._util.RemoteProtocolError: illegal header line: bytearray(b'Cookie: \

### DISCUSSION THREAD

**Discussion by MayankFawkes** [fastapi/fastapi#6453] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
sys.argv = [
			"gunicorn",
			"--preload",
			"-b", f"0.0.0.0:{port}",
			"--threads", str(multiprocessing.cpu_count()),
			"-w", str((multiprocessing.cpu_count()*2)+1),
			"--max-requests", "1000",
			"--max-requests-jitter", "50",
			"-k", " uvicorn.workers.UvicornH11Worker",
			"--keep-alive", "0",
			"main:app",
		]
```
```


### Description

i am using gunicorn with ``uvicorn.workers.UvicornH11Worker`` and keep getting the error
i found a lot solutions and non worked for me
i am using digital ocean 


### Operating System

Linux

### Operating System Details

OS: Ubuntu 20.04LTS
Python 3.8.10
Fastapi 0.70.0
Uvicorn 0.14.0
Gunicorn 20.1.0

### FastAPI Version

0.70.0

### Python Version

3.8.10

### Additional Context

![image](https://user-images.githubusercontent.com/43340190/144101577-5ca7c89e-23bc-445b-97f4-3b7b1edb9d32.png)

**Comment by omerXfaruq** [comment]
Hello @MayankFawkes , I got this error while deploying a fastapi server on digital ocean droplet.
It occurs when I try to send https requests to the server. I suspect it is because of not having SSL sertificate.

How about you, did you solve this issue?

**Comment by MayankFawkes** [comment]
@FarukOzderim  same i am also using digital ocean and it still didn't solve for me but i am just ignoring the error. I guess it is because of some bots sending https request on port 80 not sure other than that error everything is working perfectly fine, i tried to solve that but it didn't work so i just ignored the error.

**Comment by Kludex** [comment, marked as answer]
Try to install `httptools`: `pip install httptools`. Uvicorn will use that package instead of `h11`.
